lib/analytics.js

```diff
@@ -498,12 +505,46 @@ Analytics.prototype.alias = function(to, from, options, fn) {
 
 /**
  * Register a `fn` to be fired when all the analytics services are ready.
+ * TODO: document this function for the new overload behaviour.
+ *
+ * @param {Function} fn
+ * @return {Analytics}
+ */
+
+Analytics.prototype.ready = function(a, b) {
+  if (is.fn(a)) {
```

Comment:
@Rowno any suggestions for naming the parameters? I used `a`/`b` coz I couldn't think of anything better.